opds_import.py

```diff
@@ -199,12 +199,14 @@ class OPDSImporter(object):
         "No existing license pool for this identifier and no way of creating one.")
    
     def __init__(self, _db, data_source_name=DataSource.METADATA_WRANGLER,
-                 identifier_mapping=None, mirror=None, http_get=None):
+                 identifier_mapping=None, mirror=None, http_get=None,
+                 metadata_client=None
+    ):
         self._db = _db
         self.log = logging.getLogger("OPDS Importer")
         self.data_source_name = data_source_name
         self.identifier_mapping = identifier_mapping
-        self.metadata_client = SimplifiedOPDSLookup.from_config()
+        self.metadata_client = metadata_client or SimplifiedOPDSLookup.from_config()
```

Comment:
We don't currently have an end-to-end test that would have caused this code to call out to metadata.librarysimplified.org, but in my content branch I added code that did cause network requests during tests.